### PR TITLE
fix: use token denom diff #ntrn-117

### PR DESCRIPTION
**Pull request by NeverHappened**
https://hadronlabs.atlassian.net/browse/NTRN-117

### CODE REVIEW

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6328234399

**Review by pr0n00gler — APPROVED** [review]

File: contracts/lockdrop/src/contract.rs [hunk 1]
```diff
@@ -563,11 +575,29 @@ fn migrate_pair_step_1(
...
         .amount;
+    let token_denom = new_pool_info
+        .assets
+        .iter()
+        .find_map(|x| match &x.info {
+            AssetInfo::NativeToken { denom } if denom != UNTRN_DENOM => Some(denom.clone()),
+            _ => None,
+        })
+        .ok_or_else(|| StdError::generic_err("No second leg of pair found"))?;
+    //calculate amount of token (ATOM|USDC) claimed from pool
```
Comment (sotnikov-s) [inline contracts/lockdrop/src/contract.rs:596]: copy+paste leftover: it's not claimed amount, but the current amoutn

File: contracts/lockdrop/src/contract.rs [hunk 2]
```diff
@@ -563,11 +575,29 @@ fn migrate_pair_step_1(
         .as_ref()
         .ok_or_else(|| StdError::generic_err("Generator address hasn't set yet!"))?;
 
+    let new_pool_info: PoolInfoV2 = ASSET_POOLS_V2.load(deps.storage, pool_type)?;
+    let new_pool_addr = get_lp_token_pool_addr(deps.as_ref(), &new_pool_info.lp_token)?;
+    let new_pool_info: astroport::pair::PoolResponse = deps
+        .querier
+        .query_wasm_smart(new_pool_addr, &astroport::pair::QueryMsg::Pool {})?;
     //get current ntrn contract balance
     let current_ntrn_balance = deps
```
Comment (sotnikov-s) [inline contracts/lockdrop/src/contract.rs:584]: I don't really like how the `current_ntrn_balance` querying is put in the middle of `current_token_balance` calculation, that's distracting. please separate these two processes somehow

**Review by sotnikov-s — APPROVED** [review]
LGTM overall except a couple of cosmetic remarks

**Pull request merged**